<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZorkGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
color: black;
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255))
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
color: black;
background:qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.05 rgba(14, 8, 73, 255), stop:0.36 rgba(28, 17, 145, 255), stop:0.6 rgba(126, 14, 81, 255), stop:0.75 rgba(234, 11, 11, 255), stop:0.79 rgba(244, 70, 5, 255), stop:0.86 rgba(255, 136, 0, 255), stop:0.935 rgba(239, 236, 55, 255))
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="0" column="1" rowspan="3" colspan="6">
     <widget class="QTextEdit" name="textEdit">
      <property name="accessibleName">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
font: 75 20pt &quot;Constantia&quot;;
color: gold;

}</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QProgressBar" name="progressBar_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">QLabel{
color: white;
background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
font: 75 12pt &quot;Constantia&quot;;
}</string>
      </property>
      <property name="text">
       <string>HEALTH</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="7">
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
font: 75 12pt &quot;Constantia&quot;;

}</string>
      </property>
      <property name="text">
       <string>Teleport</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
background:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 75 12pt &quot;Constantia&quot;;
}</string>
      </property>
      <property name="text">
       <string>Go East</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
background:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 75 12pt &quot;Constantia&quot;;
}</string>
      </property>
      <property name="text">
       <string>Go West</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="pushButton_10">
      <property name="styleSheet">
       <string notr="true">color: white;
font: 75 12pt &quot;Constantia&quot;;</string>
      </property>
      <property name="text">
       <string>Item 2</string>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="QProgressBar" name="progressBar_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="3" colspan="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
background:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 75 12pt &quot;Constantia&quot;;
}</string>
      </property>
      <property name="text">
       <string>Go South</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel{
color: white;
background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
font: 75 12pt &quot;Constantia&quot;;
}</string>
      </property>
      <property name="text">
       <string>STAMINA</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="5">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
background:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 75 12pt &quot;Constantia&quot;;
}</string>
      </property>
      <property name="text">
       <string>Go North</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="pushButton_8">
      <property name="styleSheet">
       <string notr="true">color: white;
font: 75 12pt &quot;Constantia&quot;;</string>
      </property>
      <property name="text">
       <string>Attack</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="3">
     <widget class="QPushButton" name="pushButton_7">
      <property name="styleSheet">
       <string notr="true">color: white;
font: 75 12pt &quot;Constantia&quot;;</string>
      </property>
      <property name="text">
       <string>Take Item</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButton_9">
      <property name="styleSheet">
       <string notr="true">color: white;
font: 75 12pt &quot;Constantia&quot;;
</string>
      </property>
      <property name="text">
       <string>Item 1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">QLabel{
color: white;
background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
font: 75 12pt &quot;Constantia&quot;;
}</string>
      </property>
      <property name="text">
       <string>ENEMY HEALTH</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
